ios/Runner/pigeon.h

```diff
@@ -10,25 +10,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class Machine;
-
-@interface Machine : NSObject
```

Comment:
We should ignore this generated file under ios

Reply:
Generated files removed from repo